README.md

```diff
@@ -304,8 +304,8 @@ Following modifiers supported
     - `L` stands for last day of month (eg: `L` could mean 29th for February in leap year)
     - `W` stands for closest week day (eg: `10W` is closest week days (MON-FRI) to 10th date)
 - *Day of Week / 5th of 5 segments / 6th of 6+ segments:*
-    - `L` stands for last weekday of month (eg: `2L` is last monday)
-    - `#` stands for nth day of week in the month (eg: `1#2` is second sunday)
+    - `L` stands for last weekday of month (eg: `2L` is last tuesday)
```

Comment:
i think it is 1 based i.e Sunday = 1
> [In day-of-week, it specifies the “last day of a week.” It can also be used with another value in <day-of-week>, like “6L”, which denotes the “last Fri](https://www.baeldung.com/cron-expressions)

Reply:
Yes both exists depending what tool you use, e.g. linux crontab uses 0 for Sunday (`man 5 crontab`).
Since the `gronx` code has been using 0 or 7 for Sunday (because it checks with go's time `Weekday()` method), I fixed the readme accordingly.

Reply:
alright then (im also thinking for a feature in yet another release to make configurable for whether to use 0 based or 1 based for ordinal modifiers - but it could be overkill given the gronx is intended to be a lean/minimal lib)